Extension/src/LanguageServer/client.ts

```diff
@@ -536,6 +536,16 @@ interface DidChangeActiveEditorParams {
     selection?: Range;
 }
 
+interface GetFileIncludeDataParams
+{
+    maxDepth: number;
+}
+
+interface IncludesGraphResult
+{
+    included_files: string[];
```

Comment:
This should be includedFiles, right?

Reply:
...also I don't think the interfaces match the naming in cpptools.

Reply:
Fixed, thank you